OpenGL/Constructs/DrawElementsIndirectCommand.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Runtime.InteropServices;
 
-namespace OpenGL.Constructs
+namespace OpenGL
```

Comment:
All other classes in the Constructs folder just has the `OpenGL` namespace, so it makes sense for this one to also have the same namespace.

Reply:
Unfortunately this could cause an issue for any existing users of this code who are `using OpenGL.Constructs` or are using the fully qualified name `OpenGL.Constructs.DrawElementsIndirectCommand`.  This would be a breaking change for any users who loads the new version of the library.  I would suggest removing this change from this PR and then either creating an issue or new PR so that this change on its own can be addressed.

Reply:
Okay, I will do that later today.